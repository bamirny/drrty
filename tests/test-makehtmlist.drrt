

def int main() {
	int i;
	list[str] names;
	names =["Rania", "Trinity", "Dylan", "Richard"];

	for (i=0; i<names.length();i = i+1){
		createElement("li",names.get(i),"");
	}
	
	return 0; 
}
