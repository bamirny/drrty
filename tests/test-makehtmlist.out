<li class="">Rania</li>
<li class="">Trinity</li>
<li class="">Dylan</li>
<li class="">Richard</li>
